packages/faustwp-cli/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Just to be sure, specifying a nested ESLint config will still use the parent rules in the monorepo root as long as they are not overwritten right?

Reply:
Correct, this package should inherit all parent rules defined higher up in the project.

For future reference, if we did want a unique ESLint config for the lets say the CLI, adding
```
{
  root: true
}
```

to a nested ESLint will inform ESLint to NOT inherit those rules.

https://eslint.org/docs/latest/user-guide/configuring/configuration-files#cascading-and-hierarchy

Reply:
Awesome! Good to know for the future